app/client/actions/groupMessagesAction.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import {
+  SET_GROUP_MESSAGES,
+  FETCHING_GROUP_MESSAGES,
+  FETCH_GROUP_MESSAGES_FAILURE } from '../actions/types';
+
+
+export function getGroupMessagesForCount(groupId) {
```

Comment:
Endeavour to add JSDoc comments to describe all your functions as stated in the Javascript style that was given to you